src/index.html

```diff
@@ -1,17 +1,192 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+    onsubmit="onSubmit()"
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname" class="field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </label>
+
+        <label for="name-field" class="field">
+          Name:
+          <input
+            type="text"
+            name="name-field"
+            id="name-field"
+          >
+        </label>
+
+        <label for="age" class="field">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="0"
+            max="0"
+            id="age"
+            required
+          >
+        </label>
+
+        <label for="date-of-birth" class="field">
+          Full date of birth:
+          <input
+            type="text"
+            name="date-of-birth"
+            id="date-of-birth"
+            placeholder="mm / dd / yyyy"
+            required
+          >
+        </label>
+
+        <label for="agreement" class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            placeholder="mm / dd / yyyy"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="email-field" class="field">
+          E-mail:
+          <input
+            type="email"
+            name="email-field"
+            id="email-field"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="password" class="field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about You!</legend>
+        <div for="" class="field">
+          Do you love
+          <label for="y">
```

Comment:
```suggestion
          <label for="yes">
```
`yes` and `no` would be more meaningful than `y` and `n`